fdbclient/NativeAPI.actor.cpp

```diff
@@ -6103,30 +6114,43 @@ ACTOR Future<Optional<ClientTrCommitCostEstimation>> estimateCommitCosts(Referen
 // TODO: send the prefix as part of the commit request and ship it all the way
 // through to the storage servers
 void applyTenantPrefix(CommitTransactionRequest& req, Key tenantPrefix) {
+	VectorRef<MutationRef> updatedMutations;
```

Comment:
I realized that we can probably make this a bit more efficient by adding a line `updatedMutations.reserve(req.transaction.mutations.size());` after we create the vector. The same would be true for the conflict range vectors.